ops/aws/canary/lambda/Makefile

```diff
@@ -24,7 +24,10 @@ build: ${BIN_TARGETS}
 .PHONY: test
 test: unittests.xml
 
-unittests.xml: $(shell find ./pkg ./cmd -type f)
+pkg/test/data.tar.gz:
+	curl https://w3s.link/ipfs/bafybeihxutvxg3bw7fbwohq4gvncrk3hngkisrtkp52cu7qu7tfcuvktnq -L -o $@
+
+unittests.xml: pkg/test/data.tar.gz $(shell find ./pkg ./cmd -type f)
```

Comment:
Do we actually need this if we're now just populating with a different CID? Could we just use `cid, err := ipfs.AddTextToNodes("some random text")` like the other tests?

Reply:
Good point. I'll have to add an extra if-then in the checksum step because they'll differ but I guess that's still better than depending on the internet when testing